### ISSUE TITLE
Restricted label language is vague

### ISSUE THREAD

**Issue by CorpulentBrony** [brambleshadow4/pony-directory#4]
The README.md file says:

> Any sites which have 18+ content or would be considered offensive to certain communities MUST have a 'restricted' tag added to them.

I noticed sites such as derpibooru.org which host adult content do not contain this tag whereas marefair.org which doesn't host any 18+ content (though it is for a con intended for an 18+ audience) does have the tag.

Can the language in README.md be updated to clarify when this tag should be used? Especially since it does appear the pony.directory site is built to exclude anything tagged with restricted at the moment.

Thanks.

**Comment by brambleshadow4** [comment]
pony.directory lists all sites that I would describe as SFW/target-audience friendly, but every site is tracked + compiled into all.pony.directory. I like to keep guidelines short + simple rather than being overly prescriptive, and for 98% of sites, there's not really any debate. If someone submits a PR + I disagree with a site's restricted/unrestricted  classification, I'll just change it after merging it in + we can have a conversation about it at that point in time. Likewise, if you want to know why a site is classified a certain way, you can just ask me like you're doing now. 

derpibooru.org and most of the other image boards aren't restricted because while they do have NSFW content, you can't view it unless you create an account and then choose to look at it.

marefair.org actually used to be on the unrestricted list. It is an 18+ con, its logo is intentionally designed to look like genitalia, and the con's quite controversial within the larger community. Personally could go either way, but as soon as I got a PR to move it to restricted, I moved it. That's the side I'd rather err on anyhow.

**Comment by CorpulentBrony** [comment]
Thanks for the response!

I was thinking of submitting a PR to add some sites but wanted to be sure I knew what the expected tagging guidelines would be.

I assume I shouldn't change marefair.org to unrestricted in a PR then? As someone closely involved with the con I can verify everything you said except the part about the logo being made to resemble genitalia, it was actually designed to be an exclamation mark. The site itself will always be completely SFW. The 18+ restriction is just to simplify having panels and vendors that had 18+ content (something most pony cons have nowadays) and is a reflection of the older fandom. And as for any controversy around the con, a lot of that seems to stem from the misunderstandings of people who did not attend the con. Most of the controversial posts I've seen are either gross exaggerations or complete fabrications unfortunately.

Also, for the record, you can view NSFW stuff on derpibooru without an account or logging in by changing your filters, but your point there makes sense and I appreciate the clarification.

Given the apparent controversy I seem to engender nowadays, would you prefer I just suggest the sites to you instead of doing it as a PR?

I also noticed twibooru.org, mlp.chat, and pone.social are marked as restricted. But I'm not sure the reasons why, since there's not much on twibooru that isn't on poner.pics (not restricted), nothing on mlp.chat that isn't available on any other matrix instance, and nothing on pone.social that isn't available on equestria.social

**Comment by brambleshadow4** [comment]
That's actually good to know about derpibooru.org and a little disappointing imo. I would hope any site would put an +18 age disclaimer first. 

I personally don't care if you send me a PR or tell me via here or on a social network. I don't think anyone should view someone making a contribution to an open project as controversial. If they do, that seems like a them problem. 

Here's my thoughts on why I labeled each site as I did: 

- I put twibooru on restricted because unlike the other boorus, images rated questionable are spoiler-ed and not hidden. And since questionable are pretty much always NSFW, that's a little too accessible to put on a list of SFW sites.
- mlp.chat looks fine except that there's a direct link to iwtcits.com which is immediately NSFW. Several of your sites also got put in the restricted bucket for that reason.
- pone.social got put on it because of the phrase "horsefucker degenerates" listed in the instance's bio, and I found that a bit too spicy for the SFW classification imo. Otherwise it looks fine.
- marefair is tricky. Some people definitely view it as offensive/exclusively catering to NSFW content. I personally don't and a lot of my friends said the con was pretty normal. It's impossible to make everyone happy whatever I choose. If you change the logo to be something that can't be mistaken for genitalia, I'd feel a lot more confident moving it off restricted. Feel free to ping me if you do the con again + get a new site up and I'll take a look.
